trs_filer/app_config.yaml

```diff
@@ -0,0 +1,53 @@
+# General server/service settings
```

Comment:
Please just name this file `config.yaml`, as there are no other config files anymore (we got rid of `log_config.yaml` in FOCA).